ttnn/cpp/ttnn/operations/conv/conv2d/device/optimized_conv_op.hpp

```diff
@@ -25,6 +25,8 @@ struct OptimizedConvParallelizationConfig {
     uint32_t num_cores_c = 1;
     uint32_t per_core_out_matrix_height_ntiles = 1;
     uint32_t per_core_out_matrix_width_ntiles = 1;
+    uint32_t per_core_out_matrix_height = -1;
```

Comment:
it's a bit weird to initialize uint with -1, is this intended?

Reply:
Yeah this is incorrect. Will set it to 0.